website/docs/podman/adding-certificates-to-a-podman-machine.md

```diff
@@ -0,0 +1,102 @@
+---
+sidebar_position: 51
+title: Adding certificates to a Podman machine
+description: Adding certificates to a Podman machine
+keywords: [podman desktop, podman, certificates]
+tags: [adding-certificates]
+---
+
+# Adding certificates to a Podman machine
+
+You can add certificates from your local certificate authority (CA) or from a third-party vendor into a Podman machine. After adding these certificates, you can use them in your images:
+
+- To secure the communication channel between the running applications in your container and the external host system
+- To validate the SSL or TLS certificates provided by external services for authentication
+
+You can also use the added certificates to fix any secure connection issue. For example, you might add some certificates into your Podman machine running on Windows Subsystem for Linux (WSL) to fix an SSL issue.
+
+#### Prerequisites
+
+- You have created a [Podman machine](/docs/podman/creating-a-podman-machine).
+- On Windows: The certificates to be installed are available on an accessible server using the `curl` command.
```

Comment:
odd sentence structure, this one says "curl", the macOS instructions say "download or created".

Maybe just have one instruction? Example. "Obtained the certificate for installation (for example, certificate.pem) or something?